sdk/template/Azure.Template/src/MiniSecretsClientOptions.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.Template
+{
+    /// <summary>
+    /// The options for <see cref="MiniSecretClient"/>
+    /// </summary>
+    public class MiniSecretsClientOptions : ClientOptions
+    {
+        internal string Version { get; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MiniSecretsClientOptions"/>.
+        /// </summary>
+        public MiniSecretsClientOptions(ServiceVersion version = ServiceVersion.V7_0)
+        {
+            Version = version switch
+            {
+                ServiceVersion.V7_0 => "7.0",
+                _ => throw new NotSupportedException("Unsupported service version.")
```

Comment:
Let's change this to https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-versioning so we're all consistent.